### PR TITLE
feat : add modal component

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
add : modal componet
Screenshot from 2022-04-02 15-57-06

### CODE REVIEW

**Review by abhij1607 — COMMENTED** [review]
Very clean code | | Good class names

can be merged

**Review by Hardiegogo — APPROVED** [review]
Code is clean. Naming of the classes can be improved.
Also I recommend adding a preview link to every PR.

File: component/modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        <h1 class="heading">Modal</h1>
+        <p class="description">
+          A modal is a dialog box/popup window that is displayed on top of the
+          current page
+        </p>
+        <p class="description">Modal create with modal-container class name</p>
+        <div>
+          <div class="flex_center">
+            <!-- here the check -->
+            <div class="modal-container">
```
Comment (Hardiegogo) [inline component/modal/modal.html:57]: I feel that naming should be consistent here:
flex-center
modal-container

or
flex_center
modal_container

**Comment by ajay-mandaviya** [comment]
> Code is clean. Naming of the classes can be improved. Also I recommend adding a preview link to every PR.
it's temp dev and just a first pr , haven't deploy it so that's why it's not there

**Pull request merged**